pkgs/by-name/up/upbound/package.nix

```diff
@@ -1,32 +1,5 @@
-{ lib, buildGoModule, fetchFromGitHub }:
-
-buildGoModule rec {
-  pname = "upbound";
-  version = "0.34.0";
-
-  src = fetchFromGitHub {
-    owner = pname;
-    repo = "up";
-    rev = "v${version}";
-    hash = "sha256-npWCGR0/iXAF7efreWhpwqboXJ24lqk5TPaB7glwEFI=";
-  };
-
-  vendorHash = "sha256-f/RPyjrC5FsOjC5wnBtqEbLeVX7VeiVYFpRXcQ2lTc4=";
-
-  subPackages = [ "cmd/docker-credential-up" "cmd/up" ];
-
-  ldflags = [
-    "-s"
-    "-w"
-    "-X github.com/upbound/up/internal/version.version=v${version}"
-  ];
-
-  meta = with lib; {
-    description =
-      "CLI for interacting with Upbound Cloud, Upbound Enterprise, and Universal Crossplane (UXP)";
-    homepage = "https://upbound.io";
-    license = licenses.asl20;
-    maintainers = with maintainers; [ lucperkins ];
-    mainProgram = "up";
-  };
-}
+{
+  callPackage,
+  version-channel ? "stable",
+}:
+callPackage ./package-common.nix { inherit version-channel; }
```

Comment:
I think we can just `mv pkgs/by-name/up/upbound/package-common.nix pkgs/by-name/up/upbound/package.nix` and it should work just as well without the need for another callPackage here.

Reply:
Cool. Done 😃